broker/Gemfile

```diff
@@ -65,7 +65,7 @@ end
 # Load extra gems specified in the broker configuration file
```

Comment:
Given the weirdness we saw here I would wrap this whole thing in a begin/rescue block.  We can't afford some edge case breaking things for customers who don't need this feature.